integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -15,6 +15,8 @@ context('Add a note', () => {
     cy.task('getActionPlan', 'H4115SD')
     cy.task('stubLearnerProfile', 'G6115VJ')
     cy.task('stubLearnerProfile', 'H4115SD')
+    cy.task('stubLearnerEducation', 'G6115VJ')
```

Comment:
I needed to stub the Curious Learner Education call for quite a few tests (that on the surface are nothing to do with the Overview page), because they go via the Overview page